### PR TITLE
Add Manjaro Linux support

### PR DESCRIPTION
**Pull request by Blameying**

### CODE REVIEW

File: grub2_theme_preview/__main__.py [hunk 1]
```diff
@@ -308,13 +308,15 @@ def _grub2_ovmf_tuple():
     if the file is missing, as a 2-tuple.
     """
     _DEBIAN = 'Debian GNU/Linux'
+    _Arch_Linux = 'Arch Linux'
     distro_map = {
-        'Arch Linux': ('/usr/share/edk2-ovmf/x64/OVMF_CODE.fd', 'edk2-ovmf'),
+        _Arch_Linux: ('/usr/share/edk2-ovmf/x64/OVMF_CODE.fd', 'edk2-ovmf'),
```
Comment (hartwork) [inline grub2_theme_preview/__main__.py:313]: Would you mind turning this variable name to all-uppercase `_ARCH_LINUX` or `_ARCH` to match `_DEBIAN` right above? Thanks in advance!

**Review by hartwork — COMMENTED** [review]
That's a nice surprise, I like it! :tada: One minor detail would make it perfect, please see my comment ~above~ below.

**Review by hartwork — APPROVED** [review]
Thank you! :pray:

**Comment by hartwork** [comment]
PS: Releases 2.2.0 with this included has just hit PyPI.

File: grub2_theme_preview/__main__.py (hunk 1 above)
Comment (Blameying) [reply grub2_theme_preview/__main__.py]: Of cource not, that's my negligence!

**Pull request merged**